dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-2.3/com.ibm.websphere.appserver.jaxb-2.3.feature

```diff
@@ -19,9 +19,6 @@ Subsystem-Name: Java XML Bindings 2.3
 -features=\
   com.ibm.websphere.appserver.classloading-1.0, \
   com.ibm.websphere.appserver.optional.jaxb-2.3
--bundles=\
```

Comment:
Why are you removing these bundles in this feature.  I see that they are pulled in with internal.optional, but there is no maven coordinates settings with the internal optional feature.